shell/platform/windows/flutter_windows_engine_unittests.cc

```diff
@@ -1169,5 +1170,30 @@ TEST_F(FlutterWindowsEngineTest, ChannelListenedTo) {
   }
 }
 
+TEST_F(FlutterWindowsEngineTest, ReceivePlatformViewMessage) {
+  FlutterWindowsEngineBuilder builder{GetContext()};
+  builder.SetDartEntrypoint("sendCreatePlatformViewMethod");
+  auto engine = builder.Build();
+
+  EngineModifier modifier{engine.get()};
+  modifier.embedder_api().RunsAOTCompiledDartCode = []() { return false; };
+
+  bool received_call = false;
+
+  auto manager = std::make_unique<MockPlatformViewManager>(engine.get());
+  EXPECT_CALL(*manager, AddPlatformView)
+      .WillRepeatedly([&](PlatformViewId id, std::string_view type_name) {
```

Comment:
This should only be called once right? If so, consider:

```suggestion
      .WillOnce([&](PlatformViewId id, std::string_view type_name) {
```